File: JUnit/saucery-core/src/main/java/au/com/fullcirclesolutions/saucery/utils/UserChecker.java
package au.com.fullcirclesolutions.saucery.utils;

public class UserChecker {
    public static boolean ItIsMe() {
            return Enviro.SauceUserName().toLowerCase().equals(SauceryConstants.MY_USERNAME);
        }
}
/*
 * Copyright: Andrew Gray, Full Circle Solutions
 * Date: 10th September 2014
 */

File: XUnit/SauceryX/Util/UserChecker.cs
namespace SauceryX.Util {
    internal static class UserChecker {
        internal static bool ItIsMe() {
            return Enviro.SauceUserName.ToLower().Equals(SauceryConstants.MY_USERNAME);
        }
    }
}
/*
 * Copyright Andrew Gray, Full Circle Solutions
 * Date: 29th July 2014
 * 
 */